### DISCUSSION TITLE
Add support (ESP32) for ruuvitag BLE sensors

### DISCUSSION THREAD

**Discussion by Elwell** [arendst/Tasmota#12770] [Ideas]
Hi folks,

I've just discovered how lovely and easy the support for the (reflashed) LYWSD03MMC sensors was with tasmota32-bluetooth and wondered if the same could be done for ruuvitag support?

ESPHome seem to support them - https://esphome.io/components/sensor/ruuvitag.html
Although support for several of their protocols (see https://github.com/ruuvi/ruuvi-sensor-protocols/blob/master/broadcast_formats.md) would be Nice, I'd like "RAW v2" first
https://github.com/ruuvi/ruuvi-sensor-protocols/blob/master/dataformat_05.md

**Comment by andreas5232** [comment]
I'd be also interested in ruuvitag support for Tasmota32. Maybe we can try to implement it together? I don't have any coding knowledge about Tasmota yet.
